src/assets/allowance-module/v0.1.0/allowance-module.json

```diff
@@ -0,0 +1,717 @@
+{
+  "released": true,
+  "contractName": "AllowanceModule",
+  "version": "0.1.0",
+  "networkAddresses": {
+    "1": "0xCFbFaC74C26F8647cBDb8c5caf80BB5b32E43134",
+    "4": "0xCFbFaC74C26F8647cBDb8c5caf80BB5b32E43134",
+    "5": "0xCFbFaC74C26F8647cBDb8c5caf80BB5b32E43134",
+    "42": "0xCFbFaC74C26F8647cBDb8c5caf80BB5b32E43134",
+    "56": "0xCFbFaC74C26F8647cBDb8c5caf80BB5b32E43134",
+    "100": "0xCFbFaC74C26F8647cBDb8c5caf80BB5b32E43134",
+    "137": "0x1Fb403834C911eB98d56E74F5182b0d64C3b3b4D",
```

Comment:
Why did we not use the normal deterministic deployment for that one?

Reply:
@dasanra told me that polygon RPCs had enforced eip155 when he tried to deploy it so I decided to use safe-singleton-factory without testing the non-eip-155 one. It shouldn't affect anything

Reply:
Yes I found this blocker when trying to deploy it on Polygon before. It was enforcing me to add a chain Id, at least in December/January that I tried